tensorflow_addons/layers/multihead_attention.py

```diff
@@ -26,34 +26,26 @@ class MultiHeadAttention(tf.keras.layers.Layer):
     [Attention Is All You Need](https://arxiv.org/abs/1706.03762) which takes
     in the tensors `query`, `key`, and `value`, and returns the dot-product attention
     between them:
+    >>> mha = MultiHeadAttention(head_size=128, num_heads=12)
```

Comment:
Add an empty line above this line.

Reply:
Thanks for the review. made the changes you asked for.